src/um/versionhelpers.rs

```diff
@@ -0,0 +1,175 @@
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use shared::minwindef::{FALSE, HIBYTE, LOBYTE, WORD};
+use shared::sdkddkver::{
+    _WIN32_WINNT_VISTA, _WIN32_WINNT_WIN7, _WIN32_WINNT_WIN8, _WIN32_WINNT_WINBLUE,
+    _WIN32_WINNT_WINTHRESHOLD, _WIN32_WINNT_WINXP
+};
+use um::winbase::VerifyVersionInfoW;
+use um::winnt::{
+    OSVERSIONINFOEXW, VER_EQUAL, VER_GREATER_EQUAL, VER_MAJORVERSION, VER_MINORVERSION,
+    VER_NT_WORKSTATION, VER_PRODUCT_TYPE, VER_SERVICEPACKMAJOR, VerSetConditionMask
+};
+#[inline]
+pub fn IsWnidowsVersionOrGreater(
```

Comment:
```suggestion
pub fn IsWindowsVersionOrGreater(
```
???

Reply:
This is fixed now, although CI is failing because I'm using `_core` which I don't really understand because it works on my machine locally.